cypress/integration/login_spec.js

```diff
@@ -0,0 +1,77 @@
+import helpers from '../helpers/helpers'
+
+describe('Login', function() {
+  before(() => {
+    cy.login()
+  })
+
+  beforeEach(() => {
+    cy.visit(`${helpers.getStoryUrl(['Apps', 'Login'], 'Login')}`)
+  })
+
+  describe('Login', () => {
+    it('should load and display login view', function() {
+      cy.contains('Dieser Bereich ist privat.', {timeout: 15000})
+      cy.get('button').should('have.length', 2)
+    })
+
+    it('should not be possible to click login button', function() {
+      cy.get('[data-cy=login-form_user-input]').then(el => {
```

Comment:
Why do you use `then()` everywhere? You should be able to append `...type(...)` and all the other commands directly after `get()`?